test/e2e/apps/statefulset.go

```diff
@@ -113,13 +123,18 @@ var _ = SIGDescribe("StatefulSet", func() {
 			}
 			framework.Logf("Deleting all statefulset in ns %v", ns)
 			e2esset.DeleteAllStatefulSets(c, ns)
+
+			if scName != "" {
```

Comment:
The test should not delete the storageclass since it didn't create it

Reply:
fixed